client/main.go

```diff
@@ -142,74 +140,44 @@ func instantShareHandler() {
 			return
 		}
 		_ = resp.Body.Close()
-		fmt.Println("done")
+		log.Println("done")
 	}(clipboard.bytes)
 }
 
+func init() { log.SetFlags(0) }
+
+func init() { runtime.LockOSThread() }
+
 func main() {
 	flag.Parse()
-	runtime.LockOSThread()
 
 	menuItems := []trayhost.MenuItem{
-		trayhost.MenuItem{
```

Comment:
Nope, it's been around since Go 1.0 (and earlier) as far as I know. There was a relevant change in Go 1.5 to make it possible to do this with map keys too. See:

https://golang.org/doc/go1.5#map_literals

https://golang.org/ref/spec#Composite_literals

> Within a composite literal of array, slice, or map type T, elements or map keys that are themselves composite literals may elide the respective literal type if it is identical to the element or key type of T. Similarly, elements or keys that are addresses of composite literals may elide the &T when the element or key type is *T.